FREnctools_lib/pyfrenctools/make_hgrid/make_hgrid_wrappers.py

```diff
@@ -0,0 +1,877 @@
+import numpy as np
+import numpy.typing as npt
+import ctypes
+
+from pyfms.pyfms_utils.data_handling import (
+    setscalar_Cint32,
+    setscalar_Cdouble,
+    setarray_Cdouble,
+    setarray_Cint32,
+    set_Cchar,
+)
+
+# lib = ctypes.CDLL("./FREnctools_lib/cfrenctools/c_build/clib.so")
+
+class make_hgrid_wrappers():
+    __libpath: str = None
+    __lib: ctypes.CDLL = None
+
+    @classmethod
```

Comment:
pyfrenctools now has a new way of loading the library.  This should be modified to use the new method